src/validator/is-dateTime.ts

```diff
@@ -0,0 +1,92 @@
+import {STRule} from '../rule/rule';
+import {STRuleFn} from '../rule/fn';
+import {STRuleModifiers} from '../rule/modifiers';
+import {STRuleNode} from '../rule/node';
+import {STRuleNodeType} from '../rule/node-type';
+
```

Comment:
Per our conversation remember to update the filename `is-dateTime.ts` to `is-date-time.ts` for consistency with the project styles.